homework01/vector.h

```diff
@@ -1,5 +1,35 @@
 
 class Vector
 {
-	// TODO
+ public:
+  Vector(double x = 0., double y = 0.)
+    {
+      this->x = x;
+      this->y = y;
+    }
+  
+  bool operator==(Vector other)
```

Comment:
As discussed Thursday: this will copy the object. It is a small object, so not dramatic, still 2 or 4 times the size of a memory address, depending on the architecture.